spec/fixtures/bolt_server/environments/production/modules/bolt_server_test/tasks/init.sh

```diff
@@ -0,0 +1 @@
+whoami
```

Comment:
for all the task implemnetations it might be better to just have empty files rather than tasks that wont work (missing shebang).